src/cursor.cpp

```diff
@@ -27,6 +27,9 @@
 #include "sqlwchar.h"
 #include <datetime.h>
 
+bool ParamSetup(Cursor *cur, PyObject *sql, PyObject* original_params, bool skip_first);
+int BindAndConvert(Cursor *cur, Py_ssize_t i, PyObject *cell, ParamInfo *ppi);
```

Comment:
Two things - first I think it would be better if these were in the header file.  I get we only use it here.

More importantly, let's change `int` to `bool` since it returns true and false.